app/views/provider/admin/account/payment_gateways/braintree_blue/edit.html.slim

```diff
@@ -94,11 +94,12 @@ p
   = form.actions do
     = form.commit_button 'Save credit card'
 
+/ TODO: All is duplicated. Move this into a pack and use braintree from node_modules
 script src="https://js.braintreegateway.com/web/3.69.0/js/client.min.js"
 script src="https://js.braintreegateway.com/web/3.69.0/js/hosted-fields.min.js"
 javascript:
-  var $form = $('#new_customer');
-  var submit = document.querySelector('input[type="submit"]');
+  const form = document.querySelector('form#new_customer');
+  const submit = document.querySelector('button[type="submit"]');
```

Comment:
Maybe dumb question, but are we selecting the same element as before in here?

Reply:
Same elements just more specific selectors.